rpmlint/pkg.py

```diff
@@ -477,7 +477,8 @@ def _extract(self, dirname):
             command_str = \
                 'rpm2cpio %(f)s | cpio -id -D %(d)s ; chmod -R +rX %(d)s' % \
                 {'f': quote(str(self.filename)), 'd': quote(dirname)}
-            subprocess.run(command_str, shell=True, env=ENGLISH_ENVIROMENT)
+            subprocess.check_output(command_str, shell=True, env=ENGLISH_ENVIROMENT,
+                                    stderr=subprocess.DEVNULL)
```

Comment:
Can we have a flag to restore this? It does occasionally come in handy to see that stuff.

Reply:
Sure, I've just done that. Old output is seen with `--verbose` argument.